ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -7,7 +7,7 @@ import UIKit
 // MARK: SegmentedControl
 /// A styled segmented control that should be used instead of UISegmentedControl. It is designed to flex the button width proportionally to the control's width.
 @objc(MSFSegmentedControl)
-open class SegmentedControl: UIControl, TokenizedControlInternal {
+open class SegmentedControl: UIView, TokenizedControlInternal {
```

Comment:
any voiceover code changes we need now that we are uiview rather than uicontrol?

Reply:
VoiceOver is still working fine, says "First, button, 1 of 3"